src/lib/Components/__tests__/FilterModal-tests.tsx

```diff
@@ -0,0 +1,24 @@
+import { Theme } from "@artsy/palette"
+import React from "react"
+
+import "react-native"
+import * as renderer from "react-test-renderer"
+
+import { FilterModal } from "lib/Components/FilterModal"
+
+it("looks like expected", () => {
```

Comment:
A couple more-specific tests would be valuable for this component. Things like...

* It only renders when the `visible` prop is true
* It can be closed